nft_ingester/src/processors/accounts_processor.rs

```diff
@@ -163,6 +168,13 @@ impl<T: UnprocessedAccountsGetter> AccountsProcessor<T> {
                                 continue;
                             }
                         };
+
+                        if unprocessed_accounts.is_empty() {
+                            tokio::time::sleep(Duration::from_millis(1000)).await;
```

Comment:
Why do we sleep here for so long? 400ms is the time of a slot, so we'll have 2.5 slots closed and a lot of messages after the sleep. I'd suggest smth <100ms might be tolarable. Currently, when there are no congestions and not much interactions we would see the following p90 latency from redis receiving the message to it being processed: 
<img width="1910" alt="image" src="https://github.com/user-attachments/assets/ab05be8d-ce5d-4fb1-b97e-b6d0e276f322" />
so even 50ms will double this metric

Reply:
In the situation when we have more than one worker and there is a batch size for example 100, then the worker will select these 100 blocks and the rest will not have a message, therefore when there are too few messages in the queue, a large number of employees does not make sense. But in general we need to see with different settings how the workers will handle messages and which setting will be better for our system.